packages/backend/src/tenants/settings/model.ts

```diff
@@ -41,3 +41,26 @@ export class TenantSetting extends BaseModel {
     ]
   }
 }
+
+const TENANT_KEY_MAPPING = {
+  [TenantSettingKeys.EXCHANGE_RATES_URL.name]: 'exchangeRatesUrl',
+  [TenantSettingKeys.WEBHOOK_MAX_RETRY.name]: 'webhookMaxRetry',
+  [TenantSettingKeys.WEBHOOK_TIMEOUT.name]: 'webhookTimeout',
+  [TenantSettingKeys.WEBHOOK_URL.name]: 'webhookUrl'
+}
```

Comment:
```suggestion
} as const
```
Then we can get autocompleted `settingsObj.exchangeRatesUrl`